### ISSUE TITLE
Survival Closed Beta Test: who wants to be in?

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#5464]
Hello,
So the time has come to get some early feedback on the survival part of the game.
You should not be excited, it is still very early, I just need some feedback on what to fix and what to improve.
Please see below on a deeper dive of the current state following my own playthrough.

=> I am looking for players that are willing to book cases and discuss about possible improvements.

It is a closed beta, if you want to be in give me your steam id and I will add you to the short list, there is no specific build so it is easier for me to manage. You can get your steam id in your account panel. If for some reason you are not comfortable with sharing your steam id here you can send it to a discord mod and they will send me a mail!

![image](https://github.com/tsunamayo/Starship-EVO/assets/6604318/da2ac9fe-5c3c-49d1-887e-9d8a662861e1)

Here is how it works at first:
Phase 1: Establish an outpost with a crucible and a crafting table.
Phase 2: Generate power and make a furnace and an assembler
Phase 3: Make an hovercraft to explore more easily
Phase 4: Make a H2 farm with an electrolyser on a shore
Phase 5: Make a first spaceship.
Phase 6: Get to the asteroids and mine crystals to craft lightcruise engine. The trip time should be acceptable at 2min.
Phase 7: TO BE DONE: Get to another planet to mine yet another crystal to acquire warp techno
Phase 8: TO BE DONE: mid/late game, do missions and interact with factions.

I think it should take you less than 5 hours depending on how much time you want to spend on your first ship. I guess you could speed run it in 2h.

Early remarks and question marks (sorry it is quite big):

- At first when you dont have a hovercraft / ship yet it is painful to walk long distance to find ore. I tried to address that with the new "boost" mode  for the jetpack, but I am not satisfied
- I want to refactor the sprint key: by default you always run, and sprint key will boost instead.
- There is a lot unwanted bounce in the player locomotion, so I need to address those
- I need to add back anti-g for easier building, so jetpack will consume H2.
- I will need a ore scanner as it can painful to find some of the rarer one (cobalt...)
- I want to diversify ore spawning to male it less repetitive, for example generate caves and spawn some ore only underground.
- An help for beginner is of course needed. I dont want a full blow tutorial, I will most likely use tied the progression help with the achievements system on steam.
- Mining is fast enough, I think the ore patch are too big except for iron (as you need loads of it)
- When you leave to mine crystals on the asteroids, make sure to take a screenshot from where you come from otherwise you might get lost. Of course the solve that I will need to add a beacon block
- I want to add a research system of some sort. I want to tied that to scrapping wrecks but still not 100% sure
- I still need to balance the power generation ladder. I dont want solar panel / wind turbine to be usable on the first ship, they should be for stations not ship.
- Inventory volume also needs storage. It feels storage unit should store a bit more. I need to work on the player storage too.
- I want to add some battery in the form of items to refill hovercraft easily. (as docking is a bit overkill)
- I also want to add inventory transfer to docking port for easier refuel
- A big source of pain is landing a ship on a sloped terrain. It makes using F5 compulsory in order to ingress the ship. My previous way of solving that was through gravity field, but there is none on your starter ship, so I need to come up with a better solution.

Of course there are plenty of things that could be added to make the whole experience more compelling like food/hunger and enemy in particular. Please do not hesitate to share what you would like to see first (knowing that working on the mid-late game content will have a higher priority!)

Cheers

**Comment by Alkaliii** [comment]
`76561198164378842`

**Comment by Janssendel** [comment]
76561198971352478

**Comment by asanagisae** [comment]
`76561197996895516`

**Comment by Kaiser-Indrasil** [comment]
76561198065912752

**Comment by ProPeach** [comment]
Steam ID: 76561198120778707
Looking forward to getting started 💪

I've been playing a little mp lately, would multiplayer be in the scope of this CBT? Might be a bit tough with disappearing inventories for clients, but I can probably get some reports still

**Comment by Crimson-Artist** [comment]
steam ID: 76561198009936169

I'll throw my hat in. I've got some ideas already having messed with basically every aspect of the game but i'll save them till i've done a deep dive into survival

**Comment by SovietSponge** [comment]
Steam ID: 76561198290886541

Tried to test out the survival features by just enabling them in creative, worked decently, can't wait to see how it's in proper survival!

**Comment by Macharius69** [comment]
Bonjour Tsuna,
Je joue entre 3 et 4h par jour en moyenne, et je voulais justement me lancer une partie en survie prochainement, pour tester les nouvelles mécaniques.
Si je peux t'aider en même temps se serait avec grand plaisir,

mon ID Steam : 76561198227155500

**Comment by Tackietacktack** [comment]
Steam-ID: 76561198084098319

**Comment by Aranhanauta** [comment]
Hello, will the player start on a planet or a planetary ring in survival mode?

**Comment by Huttrem** [comment]
steam ID: 76561198991398042

Already testing some stuff, it will be amizing to be part of the early steps to have survial, i was thinking in the food part yesterday before sleep!!!

**Comment by Tisdag1** [comment]
Steam ID: 76561198053851453
I have been so excited for the survival aspect! I would love to help with the testing & feedback!

**Comment by TwoNiner29** [comment]
76561198073278478

**Comment by Asnro** [comment]
76561198020638299

I'd be excited to be able to help contribute in this!

**Comment by Jimvanhazendonk** [comment]
I'd love to test the beta out.
My account is 76561198030907166

**Comment by Briaireous** [comment]
76561198300002563

**Comment by Womble-TC** [comment]
woop woop

76561198099943577

**Comment by BigBadKangaroo** [comment]
76561199074733539

**Comment by Gallt** [comment]
76561197976368745

did a mock survival a while ago and have made small survival blueprints. Would love to help offically!

**Comment by tsunamayo** [comment]
Okay thank you all for your interest guy! I am fixing a few issues, mainly relative to the player inventory space. I keep you posted!

**Comment by Xerma** [comment]
76561198049413331

Exciting to see it come together!

**Comment by surajnairpv** [comment]
Steam ID: 76561198089555474

READY FOR ACTION.

**Comment by TIKIRobo** [comment]
while i unfortunately dont have time to test :( i did see the 
>A big source of pain is landing a ship on a sloped terrain. It makes using F5 compulsory in order to ingress the ship. My previous way of solving that was through gravity field, but there is none on your starter ship, so I need to come up with a better solution
and I thought that either magnetic boots or some sort of magnet block could work?
they would lock your player to a surface to stop you from sliding and hopefully fix the getting around ships on slopes issue (if thats the issue you're having) and most importantly to keep them balanced they wouldn't work if opposite to a gravity field (that way you can't walk on the ceilings of your planet side base or whatever 

also they could be used to fix an issue i found the last time i played, when inside a gravity field you can get "parented" to another entity's gravity which can cause issues if the ships are flying in opposite directions (generally you get pulled through a wall and into space lol) the mag boots would let you lock yourself to the floor of your ship keeping you parented to that ship's gravity field

**Comment by Uncle-Ulty** [comment]
SteamID: 76561198201070610

It's time for GOTY 2024, gogogo Tsuna!

**Comment by Zouzeug** [comment]
76561198217916987 
un honneur

**Comment by MadTheDog** [comment]
76561198114364997

**Comment by dean-proca** [comment]
76561197993533023

**Comment by tsunamayo** [comment]
Okay guys thanks for your support, I have shared the access to the survival mode with you. You will need to be online though to play, as it check an google sheet for your steam id.
Please book any bug you see, I will also open a few case to discuss about specific points. 
Thanks all!

**Comment by surajnairpv** [comment]
Hi tsuna,
how do you get into the survival beta? I have already shared my steam ID but can't figure how to access it, sorry if its a noob question .

**Comment by fellow-starmadian** [comment]
unsure if I'm too late to the party, but from what I've seen on the discord, this is looking promising, gotten me interested in hopefully finding some bugs and helping iron things out :D
steam ID: 76561198044920075

**Comment by Macharius69** [comment]
how to activate survival mode?
I can't select it in new character
my game is up to date but I don't seem to have access to the beta

**Comment by jasonfogleman** [comment]
76561197998594152

DoubleBotch on the SEVO discord. 
I'd love to help.

**Comment by tsunamayo** [comment]
> how to activate survival mode? I can't select it in new character my game is up to date but I don't seem to have access to the beta

Tu devrais voir en bas a gauche un message indiquant que l access a la beta est donne. Il faut etre en ligne pour que cela marche. de mon cote je t ai bien rajoute hier!

**Comment by rgillooly** [comment]
76561198075719588

**Comment by Macharius69** [comment]
> > how to activate survival mode? I can't select it in new character my game is up to date but I don't seem to have access to the beta
> 
> Tu devrais voir en bas a gauche un message indiquant que l access a la beta est donne. Il faut etre en ligne pour que cela marche. de mon cote je t ai bien rajoute hier!

je suis bien en ligne sur Steam, comme toujours, je suis bien en expérimental, je suis bien en 24w06b, mais pas de message concernant la Beta, et pas possible de sélectionner le mode survie...
ne perds pas de temps avec ça, je dois avoir un truc qui bloque... tans pis... je t'aiderais une prochaine fois

**Comment by walazzawe** [comment]
If it's not too late, I'd love to help out. Here is my ID:
76561198019088001

**Comment by Macharius69** [comment]
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/14315973/Player.log)
https://steamcommunity.com/profiles/76561198227155500/

j'ai copié collé, comme ça pas d'erreur possible
Merci à toi de prendre du temps pour ça

**Comment by zorozeenee** [comment]
LEZ GO

Steam ID: 76561198105815853

**Comment by Greykane93** [comment]
If it's not too late, I'd like to help out as well! I don't have tons of hours in the game yet, but I'm a software tester by trade, and would really enjoy lending a hand!

Steam ID:76561199068017476

**Comment by geri203** [comment]
Hi, can i join? My steam id:76561198045008913

**Comment by BorisLichev** [comment]
76561198067606480

**Comment by Aldruun** [comment]
I'd love to contribute something to the game, so please count me in:

76561198080586424

**Comment by animation303** [comment]
76561197986818335
 
 you are making the game I wish I could make, I would love to contribute.

**Comment by AliOsseili** [comment]
Hello friend,

76561199001562560

Bought your game just to support the dev process. Interested in helping out more while I work from home.

Cheers.

**Comment by DrizztCode** [comment]
hi Tsuna - i am currently testing out the survival features on the current experimental build (24w09b) - but i am hesitant to give any feedback, since i am sure that all of those features are under work in the beta

if you are still accepting people for the beta then please let me know, since now i am back to playing again, i will be able to play every night (and all weekend lol)

on the regular game i have been playing by putting my player on a planet, and only allowing myself to salvage scrap from wrecks - no components - with all of the limiting features turned on (player requires oxygen, limited fuel, crafting requires components, only using the jump pack)

i have currently managed to build a ship and was on my way to the asteroid ring (which will obviously take a lot longer in vanilla lol) - and then i remembered that i had failed to pack any oxygen or spare fuel

so i headed back and am currently working on getting water into the electrolyzer (think i have it sorted with a submerged fuel scoop)

anyway - pretty great updates since i last played - and a perfect feature base on which to build the survival mode

so - please do let me know if i can help - but of course i can see that many people are asking - and so if you decided to make a solid cut off line i totally understand

and either way i wish you the very best with the beta - and i will continue to report bugs with the regular builds - but may avoid too many (or any) suggestions since i am sure all of the features are being heavily worked

thanks again for all the great work :-)

**Comment by YnotnA** [comment]
Hi Tsuna,

Can i join Beta Test ?
Steam ID : 76561197998426009

**Comment by Eon09** [comment]
Hi , can i join beta test?

Steam ID: 76561198079480609

**Comment by SalmonGod** [comment]
Steam ID: 76561197961327976

Very excited for the development of this game!  You already have a better building system than Space Engineers or Empyrion.  With good survival gameplay, Starship EVO would blow the whole genre away.

**Comment by RyuriBK** [comment]
ID Steam: 76561198076869968

Im really want to join the Survival Beta test!

**Comment by moocowswag** [comment]
Steam ID: 76561198083370068


>Phase 1: Establish an outpost with a crucible and a crafting table.
>Phase 2: Generate power and make a furnace and an assembler
>Phase 3: Make an hovercraft to explore more easily
>Phase 4: Make a H2 farm with an electrolyser on a shore
>Phase 5: Make a first spaceship.


Without even playing yet, I can say this all looks fine as default progression. Its natural as each step also serves to provide the player with convenience which is important to make progression engaging to the player at such a critical point where new players will find themselves. 

I just suggest planning ahead and leaving room for wider / side progression in the far future in the form of more things that you can do on planets before your first spaceship. (But make sure to establish this default way to progress you currently have as dominant or you will risk sensory overload on new players)

This vastly increases replay-ability and opens up opportunities for entirely different scenarios sort of similar to space engineers.

For example if there is more that you are able to do/accomplish before getting your initial spaceship, it opens up the possibility of a scenario where its actually harder to get off planet without actually making that process boring, but instead interesting and fun.

There is a massive domino effect to this because now we have a reason to let planets be fundamentally different in terms of things like gravity. (Which could make getting a spaceship capable of leaving the planet harder) 

A difference in physics like this in planets allows players to decide to specialize their designs in that particular environment and helps encourage a feeling of unique territory that they hold special dominion over others.



TLDR: How you design the early game has the potential to open and close many doors when it comes to designing the entire rest of the game.

**Comment by Jimvanhazendonk** [comment]
> I'd love to test the beta out. My account is 76561198030907166

Somehow my Survival game has disappeared, and I'm not able to create a new one even thought the game states I'm in closed beta. Have I been removed from the program?
I was not active for some time. I had to focus on exams for my study.
Would it be possible to reactivate it?

**Comment by Jordanmuss99** [comment]
76561198048432515

Would Also love to join :)

**Comment by vortenian** [comment]
76561198023601181 would also love to join took me a while to locate this, thank you

**Comment by dragonion123** [comment]
76561199161184151

Would also also love to join

I have quite a lot of ideas for stuff to do in survival, and would like to test them out!

**Comment by funkyvid092** [comment]
I would like the join the beta as well if you don't mind :D
Steam ID: 76561198338798486

**Comment by TheDrunenFROG** [comment]
I would love to join this beta:
My steam ID is: 76561198066734312

**Comment by Laga121** [comment]
76561199007026851

**Comment by Barek81** [comment]
Steam-ID: 76561198037245296

**Comment by LOBOxTOMIA** [comment]
76561198014753083

**Comment by Dwarf-LordPangolin** [comment]
Hi Tsuna! Would it be possible to be involved in this? I'd love to help test things out.

Steam ID: 76561198018141136

**Comment by Jeux1d100** [comment]
Hello Tsuna, i’d love to test the beta Survival, if you invite me in.

Steam ID : 76561197992733052

Merci bien

**Comment by Vixenin** [comment]
I am not sure if is still available, but I would like to sign up for the survival play test.

My id is : 76561198128711064

Thank you in advance

**Comment by Cappuccin0w0** [comment]
hello hi I'd love to try out survival :)

Steam ID: 76561198301866575

**Comment by HeckenDiver** [comment]
Hi there, would love to give Survival a proper test! In case, aplications are still open, heres my Steam-ID: 76561197967504343

**Comment by freakxtc** [comment]
if your still looking for beta testers for survival i would be very interested. Steam ID: 76561198086362749

**Comment by Daiana-0123** [comment]
Hello, if you are still looking for testers I would like to join.
Steam ID：76561199702968569

thank you✨

**Comment by KirinSage** [comment]
Hi I don't know if you're still accepting people into the beta but if you are I'd be happy to test it and give feedback, my steam id is 76561198047587209

**Comment by assasinwar9** [comment]
my steam id is 76561198005802016 , would be happy to test and provide feedback :)

**Comment by xxSkyy** [comment]
also want survival playtest 76561198050058243

**Comment by Vandrid** [comment]
I would like to join in on the fun!
ID: 76561198021845284

**Comment by ErdanThren** [comment]
76561197996558342 - I too would like to join the beta

**Comment by tsunamayo** [comment]
So I have added all of you to the new phase. Please check the new post for what to look for! #5893

@jasonfogleman
@rgillooly
@walazzawe
@zorozeenee
@Greykane93
@geri203
@BorisLichev
@Aldruun
@Josral
@animation303
@AliOsseili
@YnotnA
@Eon09
@SalmonGod
@RyuriBK
@moocowswag
@Jimvanhazendonk
@Jordanmuss99
@vortenian
@dragonion123
@TheDrunenFROG
@Laga121
@Barek81
@PixieRaveGirl
@LOBOxTOMIA
@LucasSnSouza
@Dwarf-LordPangolin
@Jeux1d100
@Vixenin
@Cappuccin0w0
@HeckenDiver
@freakxtc
@KirinSage
@assasinwar9
@MushDropRevive
@xxSkyy
@Vandrid
@ErdanThren)

**Comment by Dwarf-LordPangolin** [comment]
Awesome! Thank you so much Tsuna! I'll get started on looking at the Second Impact stuff you linked.

**Comment by Alterdufisch** [comment]
steam ID: 1324295799

**Comment by B11TME** [comment]
Any room for another tester tsunamayo? I have been testing games for years in closed phases. Also computer technician for 25 years. Only got the game yesterday and have no idea how this one got past me lol. 99% of the games i play are survival so i would be interested in testing this for you.

Here is my steam ID

Kind regards

76561197997823994

**Comment by RaysandraRexxia** [comment]
Steam ID: 76561198010423157
I backed the Kickstarter, and have played since the beginning!

**Comment by gugupizza** [comment]
76561198042025309

**Comment by yuukiw** [comment]
Hi steam id; 76561198003881370

**Comment by czaski** [comment]
ID Steam: 76561199012184181

**Comment by Gristle0** [comment]
76561198001275487

**Comment by parda21** [comment]
Hey, 
I cant wait to try it out :) 

SteamID: 76561198361069597

**Comment by Ganakim** [comment]
If you've got room for more testers, I've got a fair bit of experience testing games. Steam ID: 76561198220155569

**Comment by Rabiator-0815** [comment]
I'd like to join. Got an invitation but the link was lost, so here is my Steam ID: 76561197962057302

**Comment by falcon470** [comment]
if its not too late 76561197971096314 game gave me an invite, looked here for it

**Comment by SydTheCat** [comment]
If it's not too late I'd really like to sign up for closed beta testing
76561198054273536
Thank you

**Comment by Fr0ntLine7** [comment]
76561198353214531 I would like to try this out, I like it so far and would love to help test.

**Comment by 777eza** [comment]
Steam ID: 76561198032163082 would love to playtest

**Comment by epicNath** [comment]
76561198289000890 Sounds really cool!

**Comment by Slyreus** [comment]
We're a bunch of merry pranksters, and we'd love to gain access to the closed beta of Starship Evo to test the multiplayer survival mode :)

Here are our Steam IDs:

- 76561198066265556
- 76561198274753545

Thanks a lot for considering our request!

**Comment by darknerciyes** [comment]
76561198107463521 i want to try

**Comment by hotweed** [comment]
Hi, sorry i'm late :) if it'is still possible,
ID: 76561198024564844
Thanks!

**Comment by Rexyboi812** [comment]
I want to join too I like finding bugs and glitches and report them occasionally.

Steam ID: 76561198005291162

**Comment by hobnob321** [comment]
Would be more than happy to try out for the next round of closed beta 
**Steam ID: 76561198250394512**

Already completed phase 1 to 5 in normal game now actually making components rather than just spawning them in 
more of a challenge in getting conveyor belts and things linked together 

All the best 
HobNob

**Comment by Auseawesome** [comment]
I would love to help out in testing the survival mode if applications are still available

Steam ID: 76561198391139525

I have previously done closed beta testing for games such as Terratech Worlds, I have experience in writing helpful bug reports and I am happy to provide any needed info to help replicate/fix bugs, issues, or gameplay nuances

**Comment by Gaetancode** [comment]
Hi there i didnt even know that a CBT exist so if i can be a part of those tester, this is my ID thx by advance !! 

Steam ID: 76561198068210248

**Comment by JRL101** [comment]
Steam ID: `76561198090685204`
Once i get access i can smash out the appropriate test run asap, especially if i can do it in one sitting. I would love to test this aspect throughly. What kind of information/feedback are you after? Anything i should keep an eye out for as a common bug(s)?
Anything you want me to break or try to break? Do i report any bypasses/tricks i find? What format would you like or prefer collected data to be in? Any conflicts you'd like tested/forced/detail scrubbed? 
Would you like a variation of tests where i break the survival sequence?

**Comment by MaxasVal** [comment]
> Hello, So the time has come to get some early feedback on the survival part of the game. You should not be excited, it is still very early, I just need some feedback on what to fix and what to improve. Please see below on a deeper dive of the current state following my own playthrough.
> 
> => I am looking for players that are willing to book cases and discuss about possible improvements.
> 
> It is a closed beta, if you want to be in give me your steam id and I will add you to the short list, there is no specific build so it is easier for me to manage. You can get your steam id in your account panel. If for some reason you are not comfortable with sharing your steam id here you can send it to a discord mod and they will send me a mail!
> 
> ![image](https://private-user-images.githubusercontent.com/6604318/303205547-da2ac9fe-5c3c-49d1-887e-9d8a662861e1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxNjE3MzAsIm5iZiI6MTcyNjE2MTQzMCwicGF0aCI6Ii82NjA0MzE4LzMwMzIwNTU0Ny1kYTJhYzlmZS01YzNjLTQ5ZDEtODg3ZS05ZDhhNjYyODYxZTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTJUMTcxNzEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmU4MWViOTJjYmMwMmIyNGM1YTIzY2M3NzM3YzMyMGUxODg2MmY3Y2Q0ZDE2NDM0YTVmNzg1MGIwYzM1NmEyYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.eTzrc_cS20Pex3XMhY0hoaa6n-HJ7yjlINutLbFCguM)
> 
> Here is how it works at first: Phase 1: Establish an outpost with a crucible and a crafting table. Phase 2: Generate power and make a furnace and an assembler Phase 3: Make an hovercraft to explore more easily Phase 4: Make a H2 farm with an electrolyser on a shore Phase 5: Make a first spaceship. Phase 6: Get to the asteroids and mine crystals to craft lightcruise engine. The trip time should be acceptable at 2min. Phase 7: TO BE DONE: Get to another planet to mine yet another crystal to acquire warp techno Phase 8: TO BE DONE: mid/late game, do missions and interact with factions.
> 
> I think it should take you less than 5 hours depending on how much time you want to spend on your first ship. I guess you could speed run it in 2h.
> 
> Early remarks and question marks (sorry it is quite big):
> 
> * At first when you dont have a hovercraft / ship yet it is painful to walk long distance to find ore. I tried to address that with the new "boost" mode  for the jetpack, but I am not satisfied
> * There is a lot unwanted bounce in the player locomotion, so I need to address those
> * I need to add back anti-g for easier building, so jetpack will consume H2.
> * I will need a ore scanner as it can painful to find some of the rarer one (cobalt...)
> * I want to diversify ore spawning to male it less repetitive, for example generate caves and spawn some ore only underground.
> * An help for beginner is of course needed. I dont want a full blow tutorial, I will most likely use tied the progression help with the achievements system on steam.
> * Mining is fast enough, I think the ore patch are too big except for iron (as you need loads of it)
> * When you leave to mine crystals on the asteroids, make sure to take a screenshot from where you come from otherwise you might get lost. Of course the solve that I will need to add a beacon block
> * I want to add a research system of some sort. I want to tied that to scrapping wrecks but still not 100% sure
> * I still need to balance the power generation ladder. I dont want solar panel / wind turbine to be usable on the first ship, they should be for stations not ship.
> * Inventory volume also needs storage. It feels storage unit should store a bit more. I need to work on the player storage too.
> * I want to add some battery in the form of items to refill hovercraft easily. (as docking is a bit overkill)
> * I also want to add inventory transfer to docking port for easier refuel
> * A big source of pain is landing a ship on a sloped terrain. It makes using F5 compulsory in order to ingress the ship. My previous way of solving that was through gravity field, but there is none on your starter ship, so I need to come up with a better solution.
> 
> Of course there are plenty of things that could be added to make the whole experience more compelling like food/hunger and enemy in particular. Please do not hesitate to share what you would like to see first (knowing that working on the mid-late game content will have a higher priority!)
> 
> Cheers

Steam -ID: 76561198016483161
Please I want to test survival :)
Thnak you

**Comment by MaxasVal** [comment]
Please add me for testing survival! :)
Steam ID: 76561198016483161

**Comment by SpaceXisLife** [comment]
Steam ID: 76561198953832821

Been Playing Stormworks and ksp for years, i stumbled across the game this week from a youtube video and fell in love at first sight! I would absolutely love to get in on the survival train and start streaming this game on twitch! I'm amazed it took me so long to find a game that so perfectly scratches my building itch.

**Comment by JRL101** [comment]
> Steam ID: 76561198953832821
> 
> Been Playing Stormworks and ksp for years, i stumbled across the game this week from a youtube video and fell in love at first sight! I would absolutely love to get in on the survival train and start streaming this game on twitch! I'm amazed it took me so long to find a game that so perfectly scratches my building itch.

Welcome! Its a hell of a game, its way more polished than a lot of alphas out there too, its hard to find ways to break the mechanics in an eventful way. If you love making intricate moving parts and detailed stuff this games perfect. The amount of moving part you can make and not crash the game is insane. You want a fold out chair, go for it. You want a mech with armor that twitches and pulses and adjusts, go for it.

**Comment by Gristle0** [comment]
Is access to the survival branch closed?

**Comment by McClamp** [comment]
I would love to help test survival. Had the game for a while and I have played periodically to keep an eye on it but I'm really waiting for survival:

Steam ID: 76561197990377112

**Comment by Voragain** [comment]
Interested in testing and bug reporting!
Steam ID: 76561198117064723

**Comment by Starbuilder2050** [comment]
this is my steam Id "76561198385467916".
I am interested in helping out with the test of the game as well as reporting bugs.

**Comment by Starbuilder2050** [comment]
maybe as a part of later gameplay with factions, you can buy blueprints for the parts for ships and base. just a suggestion with the scaping of wreaks to get blueprint fragments that can be put together once you scrap enough wrecks.

**Comment by WastedProphet** [comment]
Here's my steam ID: 76561198069526809

I'll try my best to report bugs and give feedback. Hope for the best.

**Comment by ninja44101** [comment]
this is my Steam ID: 76561198126705984

I love space games.  so I would be happy to give feedback

**Comment by kapitansen** [comment]
Steam ID: 76561198176522485

**Comment by Timnuts** [comment]
Hey, would love to give feedback!

Steam ID: 76561198040173395

**Comment by eatonjl** [comment]
Not sure if you are still letting people try this out, but here is mine if you are willing to add me.
Steam ID: 76561198060025344

**Comment by mrcomq** [comment]
steam:76561198107871588
I've been waiting for the survival mode of this game for a long time. I just heard about the beta version, I'm very excited, please include me too

**Comment by Paul-van-Hummel** [comment]
Hi, I've been playing the game for a while now, and I want to try out the survival mode to get more understanding of the game regarding resources and applying them into builds. Hope you are still adding new people in survival mode.
  
Steam ID: 76561198392618933

**Comment by ellebell43** [comment]
Steam ID: 76561198141774518
